src/core/passableResult/spec.js

```diff
@@ -33,12 +33,27 @@ describe('module: passableResult', () => {
         expect(output.hasWarnings()).to.equal(false);
     });
 
-
-    ['done', 'getErrors', 'getWarnings', 'hasErrors', 'hasWarnings'].forEach((method) => {
+    [
+        'cancel',
+        'after',
+        'done',
+        'getErrors',
+        'getWarnings',
+        'hasErrors',
+        'hasWarnings'
+    ].forEach((method) => {
         it(`Should expose ${method} method`, () => {
             const res = passableResult(faker.lorem.word());
             expect(typeof res.output[method]).to.equal('function');
         });
+
+        describe('When output is JSON stringified', () => {
+            it(`Should not expose ${method} method`, () => {
```

Comment:
Assert the positive, not the negative. Don't say what it doesn't do, state what it does do.